### DISCUSSION TITLE
Greedy-Plum46

### DISCUSSION THREAD

**Discussion by Ballaswqua** [Monika-After-Story/MonikaModDev#9312] [general]
I don't know why but the community keep
![Screenshot_20220612-235023_Reddit](https://user-images.githubusercontent.com/107364696/173245898-07d67981-75ca-4581-8e1d-22067b25ebfd.jpg)
s getting locked out for me I haven't done anything it's not saying I'm banned or anything like that can you tell me why this is happening

**Comment by SorcererPug** [comment]
Is Greedy-Plum46 your username? I'm unable to find you on Reddit. Could you please message me on Reddit? My username is Spikeran1.
